### PR TITLE
Need to split the maximum size for tipline content video files.

### PR DESCRIPTION
**Pull request by caiosba**
## Description

We need one maximum size for the Check upload and another one that is actually the maximum size supported by WhatsApp. This separation is needed because we need to convert the uploaded video to H.264, which is the codec support by WhatsApp. So we need a safe margin for that.

Fixes: CV2-5326.

## How has this been tested?

Unit test updated.

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [x] I have added regression tests, if the PR fixes a bug
- [ ] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

### CODE REVIEW

File: app/models/concerns/tipline_content_video.rb [hunk 1]
```diff
@@ -3,13 +3,18 @@
 module TiplineContentVideo
   extend ActiveSupport::Concern
 
-  # MP4 less than 10 MB (WhatsApp supports 16 MB, let's be safe)
-  def header_file_video_max_size
+  # Max size that WhatsApp supports
+  def header_file_video_max_size_whatsapp
+    16
```
Comment (melsawy) [inline app/models/concerns/tipline_content_video.rb:8]: It's better to read the value from config not hard coded

File: app/models/concerns/tipline_content_video.rb [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
+  end
+
+  # Max size for Check (we need to convert it to H.264, so let's be safe)
+  def header_file_video_max_size_check
     10
```
Comment (melsawy) [inline app/models/concerns/tipline_content_video.rb:13]: It's better to read the value from config not hard coded

**Review by melsawy — APPROVED** [review]
Just have a suggestion to read size values from config not hard coded

**Review by jayjay-w — APPROVED** [review]

File: app/models/concerns/tipline_content_video.rb (hunk 1 above)
Comment (jayjay-w) [reply app/models/concerns/tipline_content_video.rb:8]: Seconded!

Comment (caiosba) [reply app/models/concerns/tipline_content_video.rb]: Done. thanks!

File: app/models/concerns/tipline_content_video.rb (hunk 2 above)
Comment (caiosba) [reply app/models/concerns/tipline_content_video.rb]: Done. thanks!

**Comment by caiosba** [comment]
Thanks guys, I turned those into configuration keys.

**Pull request merged**